### ISSUE TITLE
Menu Changes

### ISSUE THREAD

**Issue by GeorgeBellTMH** [jesus-collective/mobile#1079]
* [x] Mobile view of menu should match desktop view
* [x] Admin page
  * [x] Edit root and subitems
  * [x] List all possible pages
  * [x] Ability to add parameters to actions
  * [x] Preview of menu for specific users or security setups
  * [x] Style it
* [ ] Permissioning of messaging icon
* [ ] Ability to configure logo
* [ ] Don’t reload data on each page refresh

┆Issue is synchronized with this [Wrike task](https://www.wrike.com/open.htm?id=829758746)

**Comment by GeorgeBellTMH** [comment]
@lucastbelem if you delete the Admin/Menu you can go to /app/admin-menu and recreate it from there...

**Comment by lucastbelem** [comment]
@GeorgeBellTMH thx, I was able to re-add it.

What do you mean by "add parameters to actions"?
How do I preview menus?
Would it be possible to add menu items that go to external urls?

Also noticed that when I edit a menu item, it goes to the end in the order of display.

**Comment by lucastbelem** [comment]
Also noticed that the profile icon is gone so I can't see my profile, or logout.

**Comment by GeorgeBellTMH** [comment]
If you look in the url for different places you can see ?create=false&id=[some-number]

The format for the box is:

{ “create”=false,”id”=“some-number” }

Basically jsonify the parameters and you can go anywhere…

There is a bug in the ordering and in the updating…

I can add functionality to go external as well...

George

**Comment by lucastbelem** [comment]
What is "props" in creating menu items?
<img width="1435" alt="Screen Shot 2022-02-06 at 9 56 15 PM" src="https://user-images.githubusercontent.com/48295718/152460985-1df83bdd-aabf-4a18-a2c1-6a2070dd8a3d.png">

**Comment by lucastbelem** [comment]
@GeorgeBellTMH testing menus. I made it so subscriptionValid can only see "Resources". It works. When a user with subscriptionValid clicks "Resources" they see all resources visible to users with subscriptionValid. However, when I click the resource itself, I see a blank screen.
<img width="1432" alt="Screen Shot 2022-02-06 at 10 20 29 PM" src="https://user-images.githubusercontent.com/48295718/152461758-55761efc-1f8a-4d27-b7ea-a4d5cc2827da.png">

**Comment by GeorgeBellTMH** [comment]
For props: if you look in the url for different places you can see ?create=false&id=[some-number]

The format for the box is:

{ “create”=false,”id”=“some-number” } - but this can change depending on which screen you are redirecting to...

Generally screens that are pluralized don't have any props...but screens that are not pluralized do (ie resources, vs resource) for resource you have to tell it which one you want...

Basically jsonify the parameters and you can go anywhere…
